test/array.asyncMap.js

```diff
@@ -3,18 +3,32 @@
 const metasync = require('..');
 const metatests = require('metatests');
 
-metatests.test('succesfull map', test => {
-  test.plan(2);
-
+metatests.test('successful asyncMap with array', test => {
   const arr = [1, 2, 3];
   const expectedArr = [2, 4, 6];
 
   metasync.asyncMap(
     arr,
-    item => item * 2,
```

Comment:
I'd prefer to leave this test the way it is, to ensure that synchronous functions are still supported.